### PR TITLE
i18n(pt-BR): Add translation for `guides/sidebar`

### PR DESCRIPTION
**Pull request by Fryuni**
#### Description

Add Portuguese translation of the sidebar navigation guide.

This is the final missing page for Portuguese translation 🎉

Portuguese is used as the internationalization example for all other languages, including the original in English, but that would not make much sense when the entire page is already in Portuguese and all the examples are assuming a site with Portuguese as the default. So I inverted those and use English as the secondary language for the examples in that section.

### CODE REVIEW

File: docs/src/content/docs/pt-br/guides/sidebar.mdx [hunk 1]
```diff
@@ -0,0 +1,541 @@
+---
+title: Navegação da barra lateral
+description: Aprenda como configurar e customizar os links de navegação da barra lateral de seu site Starlight.
+---
+
+import FileTree from '../../../../components/file-tree.astro';
+import SidebarPreview from '../../../../components/sidebar-preview.astro';
+
+Uma barra lateral bem organization é essencial para uma boa documentação por ser uma das principais maneiras que usuários vão navegar pelo seu site. Starlight provê um conjunto completo de opções para customizar o layout e conteúdo de sua barra lateral.
```
Comment (delucis) [inline docs/src/content/docs/pt-br/guides/sidebar.mdx:9]: Typo I think?

```suggestion
Uma barra lateral bem organizada é essencial para uma boa documentação por ser uma das principais maneiras que usuários vão navegar pelo seu site. Starlight provê um conjunto completo de opções para customizar o layout e conteúdo de sua barra lateral.
```

**Review by delucis — COMMENTED** [review]
Very sneaky — typo on the fifth word to make me check the full article which is totally fine otherwise 😁

Thanks @Fryuni — looks great!

File: docs/src/content/docs/pt-br/guides/sidebar.mdx (hunk 1 above)
Comment (Fryuni) [reply docs/src/content/docs/pt-br/guides/sidebar.mdx:9]: Hawk eyes

**Review by delucis — APPROVED** [review]
pt-br — complete 🫡

Thanks for all your work @Fryuni 💖

**Pull request merged**